examples/demo-configuration/atmos.yaml

```
@@ -0,0 +1,5 @@
+# Description: Configuration file for the Atmos CLI
+# default import path is ./atmos.d import .yaml files from the directory and merge them
+base_path: "./"
+
+  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)